seahorse-workflow-executor/deeplang/src/main/scala/ai/deepsense/deeplang/DOperation.scala

```diff
@@ -98,9 +102,39 @@ abstract class DOperation extends Operation
   def inferGraphKnowledgeForInnerWorkflow(context: InferContext): GraphKnowledge = GraphKnowledge()
 
   def typeTag[T : ru.TypeTag]: ru.TypeTag[T] = ru.typeTag[T]
+
+  val reportType: ChoiceParam[ReportType] = ChoiceParam[ReportType](
+    name = "report type",
+    description = Some(""))
+  setDefault(reportType, ReportParam.Extended())
+
+  def getReportType: ReportType = $(reportType)
+
+  def setReportType(value: ReportType): this.type = set(reportType, value)
 }
 
 object DOperation {
   type Id = models.Id
   val Id = models.Id
+
+
+    object ReportParam {
+      case class SchemaOnly() extends ReportType {
+        override val name: String = "Report schema only"
+        override val params: Array[Param[_]] = Array()
+      }
+
+      case class Extended() extends ReportType {
+        override val name: String = "Report schema, count and for small reports content"
+        override val params: Array[Param[_]] = Array()
```

Comment:
Those descriptions are accurate only for dataframes. However, operations are producing reports for many different doperables